readme.md

```diff
@@ -2,6 +2,7 @@ Internationalization
 ====================
 
 [![npm version](https://badge.fury.io/js/%40aboutbits%2Finternationalization.svg)](https://badge.fury.io/js/%40aboutbits%2Finternationalization)
+[![GitHub license](https://img.shields.io/github/license/aboutbits/internationalization)](https://github.com/aboutbits/internationalization/blob/master/license.md)
```

Comment:
I would name this `license` as it is named in the final image.

And I would do the same with the `npm version` -> `npm package`.